client/src/pages/ModalBookingScreen.js

```diff
@@ -98,8 +98,8 @@ function ModalBookingScreen({
 			open={open}
 			trigger={
 				<Icon.Group>
-					<Icon name="map marker alternate" inverted color="orange" />
-					<Icon corner name="add" color="orange" />
+					<Icon link name="map marker alternate" inverted color="orange" />
+					<Icon link corner name="add" color="orange" />
```

Comment:
I didn't get at the all search this link what is exactly for!! with or without functionality is the same!?